lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -400,26 +400,60 @@ export class BrowserCacheManager extends CacheManager {
      * Gets the active account
      */
     getActiveAccount(): AccountInfo | null {
-        const activeAccountIdKey = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
-        const activeAccountId = this.browserStorage.getItem(activeAccountIdKey);
-        if (!activeAccountId) {
+        const activeAccountKeyNew = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT_NEW);
+        const activeAccountValueNew = this.getItem(activeAccountKeyNew);
```

Comment:
Same comment here about "new". Maybe this?

```suggestion
        const activeAccountByLocalHomeAccountIds = this.getItem(activeAccountKeyNew);
```